build.sh

```
@@ -1,5 +1,7 @@
 #!/usr/bin
-packages="contracts common-types machine cf.js node node-provider playground dapp-high-roller"
+packages="contracts common-types cf.js machine node node-provider playground dapp-high-roller"
+
+set -e
```

usually `set -e` goes at top of file before any variables etc

---

@snario Any particular reason?

---

The default behavior of shell scripts is to ignore any errors anywhere in the execution pipeline.

Putting `set -e` explicitly says to abort if any failure happens, so putting it at the top ensures anything that comes after it is covered.

---

:+1: TIL